doc/examples/segmentation/plot_rolling_ball.py

```diff
@@ -294,3 +295,23 @@ def plot_result(image, background):
 
 plot_result(image[30, ...], background[30, ...])
 plt.show()
+
+######################################################################
+# 1D Signal Filtering
+# -----------------
+#
+# As another example of the n-dimensional feature of
+# ``rolling_ball``, we show an implementation for 1D data. Here,
+# we are interested in removing the background signal of an ECG waveform
+# in order to detect significant peaks. Smoother peaks van be removed
```

Comment:
```suggestion
# to detect significant peaks. Smoother peaks can be removed
```

Reply:
What do you mean by significant?

Reply:
```suggestion
# to detect significant peaks (higher values than the local baseline). 
# Smoother peaks can be removed
```

Reply:
Is the additional explanation for significant peaks (inside parenthesis) better? Thanks for the corrections.

Reply:
It's better. Perhaps prominent peaks is rather concise and precise.

Reply:
![screenshot](https://user-images.githubusercontent.com/20914037/120479475-48eab800-c3ae-11eb-8ec7-f303075a77aa.png)
 Hi @sciunto, sorry for the inconvenience but I applied your corrections and fix a couple of errors and then pushed to the issue5255 branch of my fork but for some reason this is not being updated in the current PR. Do you have any experience with this issue?

Reply:
That must appear automatically. Double check the branch with git branch, the remote with git remote, and the history with git log. Try to push again.